pkg/test/framework/features/features.yaml

```diff
@@ -30,9 +30,16 @@ features:
   # features that allow users to secure their services and service mesh.
   security:
     peer:
+      secure-naming:
     user:
     control-plane:
       k8s-certs:
+      mtls:
+      tls:
+      plugin-cert:
+    webhook:
+    chiron:
+    vault:
```

Comment:
not sure this should be top level. something like extensibility or external CAs or something maybe?

Reply:
it is no need anymore remove it